KeenClient/KIODBStore.m

```diff
@@ -50,17 +54,27 @@ @implementation KIODBStore {
 - (instancetype)init {
     self = [super init];
 
-    if(self) {
+    if (nil != self) {
```

Comment:
Yeah I thought it was the same. I too prefer explicit checks in C-ish languages, but some people who are used to JS or Python have a conniption seeing that. Whatever you choose is fine with me as long as we just be consistent going forward.